web/src/CityContentSwitcher.tsx

```diff
@@ -108,9 +107,8 @@ const CityContentSwitcher = ({ languageCode }: CityContentSwitcherProps): ReactE
                 languageCode={languageCode}
                 isLoading
                 city={city}
-                pageTitle={null}>
-                <LoadingSpinner />
-              </CityContentLayout>
```

Comment:
❓  why you didn't use a `PageSkeleton` here instead ?

Reply:
because there is no need for this, we are anyway showing skeleton in necessary components :thinking: